src/styles.css/blocks/card.css

```diff
@@ -0,0 +1,105 @@
+.card {
+  display: block;
+  padding: 16px;
```

Comment:
you can add more `padding-top` to your card and not move your `img` with additional margin